### DISCUSSION TITLE
Termux

### DISCUSSION THREAD

**Discussion by Heringen** [ORelio/Minecraft-Console-Client#2359] [Q&A] [answered]
How would one go with using MCC via termux on adroid 6.0?
i found https://github.com/MCCTeam/Minecraft-Console-Client/issues/1000 this guide however its outdated

**Comment by milutinke** [comment, marked as answer]
> How would one go with using MCC via termux on adroid 6.0? i found #1000 this guide however its outdated

https://mccteam.github.io/guide/installation.html#run-on-android

**Reply by Heringen** [reply]
Im havinf problem with net 6.0.
I entered everything up to 
export DOTNET_ROOT=/root/.dotnet/
export PATH=$PATH:$DOTNET_ROOT:$DOTNET_ROOT/tools
and it works fine, but when i type dotnet after that it says its directory doesnt exit

**Reply by milutinke** [reply]
> Im havinf problem with net 6.0. I entered everything up to export DOTNET_ROOT=/root/.dotnet/ export PATH=$PATH:$DOTNET_ROOT:$DOTNET_ROOT/tools and it works fine, but when i type dotnet after that it says its directory doesnt exit

Check out the Microsoft documentation for your distro, the installation path might be different.

**Reply by Heringen** [reply]
What do you mean by that?(im not too much into such stuff)

**Reply by milutinke** [reply]
> What do you mean by that?(im not too much into such stuff)

You will have to open up this https://learn.microsoft.com/en-us/dotnet/core/install/linux and find instructions for your distro, if they do not work, try with the manual installation.

**Reply by Heringen** [reply]
Isn't that whole process pretty much covered in https://mccteam.github.io/guide/installation.html#run-on-android tho?

**Reply by milutinke** [reply]
> Isn't that whole process pretty much covered in https://mccteam.github.io/guide/installation.html#run-on-android tho?

Have you tried running the binary of dotnet directly from it's folder?
Also, have you added the export variables to `.bashrc` file?

**Comment by ghost** [comment]
This might help https://github.com/678435021/mcc-on-termux
